seed/views/v3/datasets.py

```diff
@@ -0,0 +1,374 @@
+# encoding: utf-8
+"""
+:copyright (c) 2014 - 2020, The Regents of the University of California, through Lawrence Berkeley National Laboratory (subject to receipt of any required approvals from the U.S. Department of Energy) and contributors. All rights reserved.  # NOQA
+:author
+"""
+
+import logging
+
+from django.http import JsonResponse
+from django.utils import timezone
+from rest_framework import status
+from rest_framework import viewsets
+from rest_framework.decorators import action
+
+from seed.data_importer.models import ImportRecord
+from seed.decorators import ajax_request_class, require_organization_id_class
+from seed.lib.superperms.orgs.decorators import has_perm_class
+from seed.lib.superperms.orgs.models import Organization, OrganizationUser
+from seed.models import obj_to_dict
+from seed.utils.api import api_endpoint_class
+from seed.utils.time import convert_to_js_timestamp
+
+from seed.utils.api_schema import AutoSchemaHelper
+
+_log = logging.getLogger(__name__)
+
+
+class DatasetSchema(AutoSchemaHelper):
```

Comment:
Let me know if there's anything notable changed outside of this class, the AutoSchemaHelper import, and the application of this DatasetSchema class to the -ViewSet (`swagger_schema=...`). 

I assumed these were the only "real" changes this file and mostly everything else was just copied over.

Reply:
Yes exactly, just wanted to follow the AutoSchmaHelper pattern but mostly everything was just copied over 👍 .